modules/mono/mono_gd/gd_mono.cpp

```diff
@@ -262,9 +262,9 @@ void GDMono::initialize() {
 		// Everything is fine with the api assemblies, load the project assembly
 		_load_project_assembly();
 	} else {
+#ifdef TOOLS_ENABLED
```

Comment:
AFAIK the `core_api_assembly*` checks would still be needed when `! TOOLS_ENABLED`.

Also it would be good to use more explicit commit logs, as we can't know what "22751" is about when reading the git log locally. It's good to mention it so that it's clickable on GitHub, but the commit title should also state what it does in plain words.